odl/util/numerics.py

```diff
@@ -836,6 +838,162 @@ def zscore(arr):
     return arr
 
 
+def simulate_slicing(shape, indices):
+    """Simulate slicing into a Numpy array with given indices.
+
+    This function is intended to simulate indexing of a Numpy array
```

Comment:
What does "simulate" mean here?

Reply:
Something like "pretending to do it, but really only looking at the resulting shape and stuff".